app/page/template/content/preferences-device-details.htm

```diff
@@ -8,13 +8,16 @@
           <span data-bind="l10n_text: z.string.preferences_devices"></span>
         </header>
         <div class="preferences-devices-model" data-bind="text: device().model" data-uie-name="device-model"></div>
-        <div class="preferences-devices-id"><span data-bind="l10n_text: z.string.preferences_devices_id"></span><span data-bind="html: z.util.print_devices_id(device().id)" data-uie-name="device-id"></span></div>
+        <div class="preferences-devices-id">
+          <span data-bind="l10n_text: z.string.preferences_devices_id"></span>
+          <span data-uie-name="device-id" data-bind="foreach: z.util.zero_padding(device().id, 16).match(/.{1,2}/g)"><span class="device-id-part" data-bind="text: $data"></span></span>
+        </div>
         <div class="preferences-devices-activated">
-          <div data-bind="html: activated_on"></div>
-          <div data-bind="html: activated_in"></div>
+          <div data-bind="foreach: activated_on"><span class="preferences-devices-activated-bold" data-bind="text: $data">?</span></div>
+          <div data-bind="foreach: activated_in"><span class="preferences-devices-activated-bold" data-bind="text: $data">?</span></div>
```

Comment:
If you look at the HTML this generated we are now adding a third empty and unneccessary span to the tree.